src/samui/frontend/src/routes/Dashboard/Alerts.tsx

```diff
@@ -37,7 +38,21 @@ class Alerts extends React.PureComponent<AlertsProps> {
             !currentRule ? (
               'Alerts Dashboard'
             ) : currentRule.savedBody ? (
-              currentRule.title
+              <div>
+                <Input
+                  id="title_input"
+                  style={{width: 300}}
+                  defaultValue={currentRule.title}
+                  onChange={e => (currentRule.newTitle = e.target.value)}
```

Comment:
you shouldn't store state by mutating a prop of a PureComponent -- it might work because of how this version of redux is implemented and how we use it, but there is no guarantee that it will keep working if we update or won't act in ways that are super unexpected (e.g. user's typing being regularly discarded if we add a polling heartbeat to update the redux state asynchronously).

Reply:
This has been addressed!